services/mlmodel/tflitecpu/tflite_cpu.go

```diff
@@ -222,6 +224,12 @@ func (m *Model) Metadata(ctx context.Context) (mlmodel.MLMetadata, error) {
 	for i := 0; i < numOut; i++ { // for each output Tensor
 		outputT := md.SubgraphMetadata[0].OutputTensorMetadata[i]
 		td := getTensorInfo(outputT)
+		if (strings.Contains(td.Name, "category") || strings.Contains(td.Name, "probability")) &&
+			m.conf.LabelPath != nil {
+			td.Extra = map[string]interface{}{
```

Comment:
the gofmt on this off

Reply:
fixed.

Reply:
Also added back a parens because I needed it!